ram_msgs/AdditiveManufacturingPose[] poses
---
string error
ram_msgs/AdditiveManufacturingPose[] selection

